api/axfeat/Cargo.toml

```diff
@@ -28,6 +28,7 @@ alloc-slab = ["axalloc/slab"]
 alloc-buddy = ["axalloc/buddy"]
 paging = ["alloc", "axhal/paging", "axruntime/paging"]
 tls = ["alloc", "axhal/tls", "axruntime/tls", "axtask?/tls"]
+dma = ["paging", "axruntime/dma"]
```

Comment:
To make the feature system works, you should also add the `dma` feature in `axstd` and `arceos_api`, and add APIs like `ax_alloc_coherent` to `arceos_api::mem`.

Reply:
I am not familiar with this part, and may take a lot time for review, you can add them by another commit. Drivers do not need these api.

Reply:
Ok, please invite me to modify your personal repository.

Reply:
done